2007-07-27  Thomas Schwinge  <tschwinge@gnu.org>

        * Makefile (CFLAGS): Add ``-fno-strict-aliasing''.

--- pfinet/Makefile	2002/09/17 20:42:06	1.35
+++ pfinet/Makefile	2007/07/27 12:39:15	1.36
@@ -95,6 +95,10 @@
 	    -I$(srcdir)/glue-include		\
 	    -I$(srcdir)/linux-src/include
 
+# Don't ask...  We use Linux code.  The problem was first noticed when
+# compiling `pfinet' with GCC 4.2.
+CFLAGS += -fno-strict-aliasing
+
 asm/checksum.h: ../config.status
 	mkdir -p $(@D)
 	echo > $@.new \
